manifests/g/GIGABYTE/GigabyteControlCenter/24.12.06.01/GIGABYTE.GigabyteControlCenter.installer.yaml

```diff
@@ -19,5 +19,9 @@ Installers:
 - Architecture: x64
   InstallerUrl: https://download.gigabyte.cn/FileList/Utility/GCC_24.12.06.01.zip
   InstallerSha256: 3A7BC03B6154E221DFD0B0E1E6847C2B65DB549817530D88B0F1C8FFC74224E1
+  AppsAndFeaturesEntries:
+  - DisplayName: GIGABYTE Control Center 24.12.06.01
+    DisplayVersion: 24.12.06.01
```

Comment:
It should be `24.10.28.01`:

![CleanShot 2024-12-24 at 03 05 45](https://github.com/user-attachments/assets/1edd3757-9a07-42f7-b08d-19df9412e635)